src/test/java/uk/gov/pay/api/service/CreatePaymentServiceTest.java

```diff
@@ -75,9 +76,9 @@ public void testCreatePayment() {
         assertThat(payment.getAmount(), is(100L));
         assertThat(payment.getReference(), is("a reference"));
         assertThat(payment.getDescription(), is("a description"));
-        assertThat(payment.getEmail(), is(nullValue()));
+        assertThat(payment.getEmail(), is(Optional.empty()));
```

Comment:
Could you remove unused `org.hamcrest.core.IsNull.nullValue` import as well, thanks